Runtime 23.08

bbhtt opened the pull request:
@zorgiepoo If you can review 🙏🏼


In the Makefile you're copying the game binary to `/app/bin` before creating the directory, it'll fail as nothing creates that directory here. You should create it first.

glew is installed in `/app`, gcc will fail to find it as it searches in `/usr`, you should probably use pkg-config to search for them.

SDL 2 is included in the runtime along with libdecor.

bbhtt committed:
Bump runtime to 23.08

bbhtt committed:
Remove SDL2 as it is in runtime

bbhtt committed:
Fix makefile

glew is installed in `/app` but gcc cannot find it there

The binary is copied before creating the directory /app/bin

zorgiepoo commented:
What version of SDL2 does it include? Historically I included SDL2 so I fix it to a specific version I built/needed, not because it wasn't already included in the runtime. I updated [my Makefile](https://github.com/zorgiepoo/Sky-Checkers/commit/99c977f51b44a8ee83c0bcc801cacb52d0f1a1b9) but may take in these changes temporarily until a future release. Maybe I should just remove hardcoding SDL2 too..

bbhtt commented:
> What version of SDL2 does it include? Historically I included SDL2 so I fix it to a specific version I built/needed, not because it wasn't already included in the runtime. I updated [my Makefile](https://github.com/zorgiepoo/Sky-Checkers/commit/99c977f51b44a8ee83c0bcc801cacb52d0f1a1b9) but may take in these changes temporarily until a future release. Maybe I should just remove hardcoding SDL2 too..

2.28.2 https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/1533f8d0393984f5684985c8b710e9fbaf9c8734/elements/components/sdl2.bst#L81 I think if you need patching or specific compile args 
it is fine to keep/add but I didn't see any being used.

> I updated [my Makefile](https://github.com/zorgiepoo/Sky-Checkers/commit/99c977f51b44a8ee83c0bcc801cacb52d0f1a1b9) but may take in these changes temporarily until a future release.

Thanks

zorgiepoo commented:
OK, I'll remove it then. Thanks.
